ynr/apps/elections/views.py

```diff
@@ -256,13 +256,16 @@ def form_valid(self, form):
                 # If we're locking this, then the suggested posts
                 # can be deleted
                 ballot.suggestedpostlock_set.all().delete()
+                ballot.mark_uncontested_winners(
+                    request=self.request, ip_address=get_client_ip(self.request)
+                )
             else:
                 suffix = "-unlock"
                 pp = "Unlocked"
+                ballot.unlock_ballot(suffix, pp)
```

Comment:
The latest test that is failing is because of the arguments you are passing here. Take a look at the `uncontested` method you added - it expects a request, and IP address, but here we are passing in two stings. So we need to update this line to pass in the same args as we do on line 260:
```
ballot.unlock_ballot(
    request=self.request, ip_address=get_client_ip(self.request)
)
```